step.yml

```diff
@@ -52,11 +52,12 @@ inputs:
 - project_path: $BITRISE_PROJECT_PATH
   opts:
     title: Project path
-    summary: Xcode Project (`.xcodeproj`) or Workspace (`.xcworkspace`) path.
```

Comment:
Suggestion: I think we should make a new input called `Package.swift` which will be under the group `SPM experimental/beta`.

The will be some context to say this is experimental/beta stage and will be subject to change/break but we love to get your feedback

Reply:
I see your point, but I'd rather lean towards the simpler solution (reuse the `project_path` input). Having a new input has many drawbacks and for me, the benefits don't justify these drawbacks:

1. If there are two inputs, we need to handle the case when both of them are defined at the same time. Which one gets the priority? Also, `project_path` is assigned a value automatically by the scanner, so it would be confusing to users to see one input not having any effect on the step behavior because another input has the higher priority.
2. What happens when we mark this feature stable? Remove the experimental input and merge it with the existing `project_path` input? That's a breaking change. Or do we keep both inputs, and just remove the experimental label? Then it will haunt us for years and confuse our users.
3. It's not yet proven that users will find this new step input and the experimental input category while having too many step inputs is a known problem that we keep hearing from users.

Reply:
Great discussion team!

My take is that this is something you can solve with documentation, rather than API. 
While a dedicated API would be nice, I'm not aware of a way we can make yaml fields like this mutually exclusive. This forces us to deal with edge cases like @ofalvai described. 

I like the idea of gathering user feedback, is there a precedent set for this, or would it be something new?